chemreg/substance/models.py

```diff
@@ -61,9 +62,34 @@ class Substance(CommonInfo):
         substance_histories (QuerySet): One to Many Substance history resources (not implemented yet)
     """
 
+    preferred_name_regex = "^[a-zA-Z0-9 =<>\\-:.,^%&/{}[\\]()+?=]{3,}$"
+    display_name_regex = "^[a-zA-Z0-9 =<>\\-:.,^%&/{}[\\]()+?=]{3,}$"
```

Comment:
should we use the RegexValidator to replace validate_casrn_format here??

Reply:
That's a good idea.  I figured CAS-RN was going to be a ubiquitous field so I spun it out into it's own thing, but seeing as the checksum and format have to be split anyways it makes just as much sense to use the RegexValidator.

Reply:
cool, I just moved the regex string into a variable like you had done on other fields. Thanks